superset/db_engine_specs.py

```diff
@@ -114,23 +114,31 @@ def get_limit_from_sql(cls, sql):
                 (?ix)          # case insensitive, verbose
                 \s+            # whitespace
                 LIMIT\s+(\d+)  # LIMIT $ROWS
-                ;?             # optional semi-colon
-                (\s|;)*$       # remove trailing spaces tabs or semicolons
+                .*$            # everything else
```

Comment:
I think this regex will fail for queries like (albeit being somewhat unusual):

SELECT ‘LIMIT 10’